### ISSUE TITLE
Item selected state not announced by screen readers in Dropdown when expanded and navigated

### ISSUE THREAD

**Issue by migpietropaolo** [microsoft/fluentui#22086]
### Environment Information

- **Package version(s)**: 8.61.0
- **Browser and OS versions**: 99.0.1150.36
- **Screen Reader**: NVDA 2021.3.3
### Describe the issue:

Screen readers are not announcing the state of a selected Dropdown item when expanding the dropdown. 

### Please provide a reproduction of the issue in a codepen:

Open NVDA and navigate to https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/dropdown.
Go to any dropdown (simple or multi-select) and select an item.
When expanding the dropdown again and navigating through the items, those that are not selected have their "not selected" status read, but those that are selected won't have that state announced.

![image](https://user-images.githubusercontent.com/91968449/158167932-4f06f696-32cf-464b-a4d2-a835856fa614.png)

#### Actual behavior:

Screen reader is not announcing the status of the selected item.

#### Expected behavior:

Screen reader should announce the status of selected and not selected items.

### Documentation describing expected behavior

[4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.

**Comment by smhigley** [comment]
This is a common misunderstanding of how NVDA and JAWS present the selected state of options. Rather than read "selected" on selected options, they read "not selected" on unselected options (as shown in the speech history in your screenshot, actually). The reason for this dates back to selection following focus in early Windows UI.

It ultimately boils down to a screen reader UX decision. The selected state is programmatically correct, and it _is_ being presented. Just not in the way you're expecting :)

**Comment by migpietropaolo** [comment]
Thank you for the clarification, Sarah!
